src/documents/FrameworkConcepts.md

```diff
@@ -20,22 +20,25 @@ consist of only the following types: `Array`, `Object`, `String`, `Number` and `
 
 ## EL Paths
 
-In some parts of the framework, we refer to "EL expressions". This is a somewhat historical phrase that perhaps sounds like it says more than it is trying to. 
-All we mean by EL expressions are dot-separated paths built from names &#8212; for example if you had defined an object `zar` with a member `boo` which has a member `baz`, 
+In some parts of the framework, we refer to "EL paths". This is a somewhat historical phrase that perhaps sounds like it says more than it is trying to. A more accurate name for "EL paths" would be "Model paths". 
+These paths are dot-separated strings built from names &#8212; for example if you had defined an object `zar` with a member `boo` which has a member `baz`, 
 you could access the nested JavaScript property by writing the expression `zar.boo.baz`. 
-If it were held in a string value, this would become an EL expression &#8212; that is, the string `"zar.boo.baz"` is an EL expression which designates the same piece of data we just referenced. 
+If it were held in a string value, this would become an EL path &#8212; that is, the string `"zar.boo.baz"` is an EL path which designates the same piece of data we just referenced. 
 The framework includes machinery for interpreting such expressions held in strings rather than at the JavaScript language level. 
 This is useful because it abstracts references to pieces of data from the actual data itself &#8212; and allows these references to be stored in documents separately from a running program. 
 It is possible to replace one object tree with another, but still to maintain a stable reference to the same subproperty `baz`, whoever it happens to be today. 
 This is particularly important in web applications where data claiming to be "your data" can suddenly arrive from anywhere (a JSON feed, some persistence, 
 a particularly aggressive version management system, etc). However it got here, you know it is your data because it is at the right path.
 
-EL expressions within Infusion can be evaluated by the framework utilities `fluid.get()` and `fluid.set()`, and also global functions can be similarly 
-invoked by `fluid.invokeGlobalFunction()`. EL path expressions of this sort are fundamental to Infusion's model-oriented thinking, and the operation of the Infusion [ChangeApplier](ChangeApplier.md).
+EL paths within Infusion can be evaluated (dereferenced with respect to a particular model) by the framework utilities `fluid.get()` and `fluid.set()`, and also global functions can be similarly 
```

Comment:
These function references could now be links to the CoreAPI page, couldn't they?